src/components/forms/verify-two-fa-code-form.tsx

```diff
@@ -85,78 +86,40 @@ const VerifyTwoFACodeForm: React.FC<{
         maxLength={6}
         autoFocus={true}
       >
-        <InputMask>
-          <span />
-          <span />
-          <span />
-          <span />
-          <span />
-          <span />
-        </InputMask>
+        <div className="flex justify-between w-96 lg:w-4/6 absolute top-0 my-8">
+          <span className="bg-background border border-gray-dark rounded h-16 w-14 lg:w-16" />
+          <span className="bg-background border border-gray-dark rounded h-16 w-14 lg:w-16" />
+          <span className="bg-background border border-gray-dark rounded h-16 w-14 lg:w-16" />
+          <span className="bg-background border border-gray-dark rounded h-16 w-14 lg:w-16" />
+          <span className="bg-background border border-gray-dark rounded h-16 w-14 lg:w-16" />
+          <span className="bg-background border border-gray-dark rounded h-16 w-14 lg:w-16" />
+        </div>
       </MultipleInputsMasked>
-      <Button className="btn btn-primary" type="submit">
+      <Button className="btn btn-primary mb-0" type="submit">
         {formatMessage({ id: "confirm" })}
       </Button>
-    </ExtendedStyledForm>
+    </Form>
   );
 };
 
-const ExtendedStyledForm = styled(Form)`
-  padding: ${({ theme }) => theme.spaces.xs} 0 0 0;
-
-  p {
-    margin-bottom: ${({ theme }) => theme.spaces.xs};
-  }
-
-  button {
-    margin-bottom: 0;
-  }
-`;
-
 const MultipleInputsMasked = styled(InputText)`
```

Comment:
Why do we keep this one?

Reply:
As I said in the description, before be able to get rid of this piece of styled component, we first need to migrate the inputs (here `InputText`) and maybe to refacto them a bit 🙂  Since I don't want to do big PRs, but rather small ones, I migrate all that was related to forms and easy to change. Inputs will come in a separate one, to avoid being blocked during several hours/days on big changes and risking to break something along the way.